x-pack/plugins/alerting/server/rules_client_conflict_retries.test.ts

```diff
@@ -141,9 +141,9 @@ async function enable(success: boolean) {
     return expectConflict(success, err);
   }
 
-  // a successful enable call makes 2 calls to update, so that's 3 total,
+  // a successful enable call makes 1 calls to update, so that's 3 total,
   // 1 with conflict + 2 on success
-  expectSuccess(success, 3);
+  expectSuccess(success, 2);
```

Comment:
For clarity, the one less call is the call to create the task manager doc (it was already created)?  At first I was confused, why are we updating the rule 3 times, and how are we doing it LESS now in this PR?  But I think this check must count the TM SO updates as well ... just checking.

Reply:
Correct, there is one less call to update the SO now. Updated the comment in https://github.com/elastic/kibana/pull/139826/commits/3bd7a3a00798a2936f71d8ab8918bbf1718fad04